README.md

```diff
@@ -54,8 +54,20 @@ Finally, CIL can be run via a Jupyter Notebook enabled Docker container:
 docker run --rm --gpus all -p 8888:8888 -it ghcr.io/tomographicimaging/cil:latest
 ```
 
-GPU support requires [`nvidia-container-toolkit`](https://github.com/NVIDIA/nvidia-container-toolkit) and an NVIDIA GPU.
-Omit the `--gpus all` to run without GPU support.
+> [!NOTE]
+> GPU support requires [`nvidia-container-toolkit`](https://github.com/NVIDIA/nvidia-container-toolkit) and an NVIDIA GPU.
+> Omit the `--gpus all` to run without GPU support.
+
+<!-- <br/> -->
+
+> [!IMPORTANT]
+> Folders can be shared with the correct (host) user permissions using
+> `--user $(id -u) --group-add users -v /local/path:/container/path`.
```

Comment:
Is `/local/path:/container/path` the exact "wording" that users should use or are they replacing it with their own paths? I think this could be made clearer

Reply:
```suggestion
> `--user $(id -u) --group-add users -v /local/path:/container/path`
> where `/local/path` is an existing directory on your local (host) machine which will be mounted at `/container/path` in the docker container.
```